api/v2/api.go

```diff
@@ -257,21 +254,46 @@ func (api *API) Close() {
 	}
 }
 
-// TLSConfig is used to enable TLS on the API service
-type TLSConfig struct {
+// TLSFiles is used to enable TLS on the API service
+type TLSFiles struct {
```

Comment:
why change this from `TLSConfig`?

Reply:
That was a straggler from when I was first experimenting with the TLS cipher configuration. I originally instantiated the http server as:
```Golang
server := &http.Server{
    Addr: ....
    Handler: api.r,
    TLSConfig: ....
```

Which confusing the compiler. Now that the TLS cipher configuration is being setup different then when I first tried, I'll switch it back.